samples/subsys/shell/fs/boards/rcar_h3ulcb_ca57.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2023 EPAM Systems
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=8192
```

Comment:
I believe you don't need fc_heap_size param anymore. As I've mentioned here https://github.com/xen-troops/zephyr/pull/67#discussion_r1266345135 "you had to set CONFIG_FS_LITTLEFS_FC_HEAP_SIZE in config because you are using cachesize = 0 in lfs_data". Now you have set cachesize so i think this param can be removed.

Reply:
Actually, the sample crashes without this param

Reply:
Ok.